### ISSUE TITLE
[SF_ACC] [List] "Contains deletable items" is being read by screen readers even when there are no items

### ISSUE THREAD

**Issue by bonniegso** [SAP/ui5-webcomponents#7601]
### Bug Description

The List component contains the invisible text "Contains deletable items", which is being read by screen readers, even if there is no data. This invisible text area should be empty if there is no data.

### Affected Component

List

### Expected Behaviour

The list should not read "Contains deletable items" when there are no items in the list.

### Isolated Example

https://zk75y5.csb.app/

### Steps to Reproduce

1. Turn on JAWS Screenreader
2. Open example in a browser
3. Using your keyboard, navigate to the empty list
4. Observe that "Contains deletable items" is read

### Priority

Medium

### UI5 Web Components Version

1.14.5

### Browser

Chrome

### Operating System

MacOS 13.5.2

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by plamenivanov91** [comment]
Hello @bonniegso,

The text is read if a List item (child of List in Delete mode) gets focused.

If you remove all of the ListItems you shouldn't be able to get to a scenario where the text is read.

Can you provide a snippet to see your app structure and how the text is read with no items in the List?

Regards,
Plamen

**Comment by bonniegso** [comment]
Hi @plamenivanov91 ,

Here is a sample where you can reproduce the issue. It is similar to how we have it in our app.
https://codesandbox.io/s/zk75y5

Best,
Bonnie

**Comment by elenastoyanovaa** [comment]
Hello @SAP/ui5-webcomponents-topic-p ,

The label text "Contains deletable items" which is added when a list is in delete mode is being rendered also when there are no available items. In reading mode this would confuse the users when navigating to read the page content as the text insists that there are items in the list. As a reference this text is added on list item level in UI5 List and the issue does not occur when there are no items.

Maybe a check whether there are items should be added in the ariaLabelModeText method of the list should be added.

Kind Regards,
Elena
